plugins/torrent/deluge_

```diff
@@ -396,4 +413,4 @@ if len(sys.argv) > 1:
     else:
         fetch_info(get_mode())
 else:
-    fetch_info(get_mode())
+    fetch_info(get_mode())
```

Comment:
Please do not remove the final newline (by convention a text file ends with the *end of line* marker).

Reply:
Yes that was my editor that wasn't configured correctly. Sorry, and fixed!